### ISSUE TITLE
Doesn't work on Gnome 3.22

### ISSUE THREAD

**Issue by francoism90** [jonnylamb/shell-hide-legacy-tray#7]
Error loading the extension, hope you can take a look. :)

Thanks.

**Comment by rlaboiss** [comment]
Indeed, this extension does not work out of the box.  I succeded to make it work with Gnome 3.22.1 by doing the following.  First, I installed the package from the [extension web page](https://extensions.gnome.org/extension/967/hide-legacy-tray/).  Then I did:

``` shell
git clone http://github.com/jonnylamb/shell-hide-legacy-tray/
cd shell-hide-legacy-tray/
cp schemas/* ~/.local/share/gnome-shell/extensions/hide-legacy-tray@shell-extensions.jonnylamb.com/schemas/
```

and I restarted gnome-shell with Alt-F2 and `r`.

No idea why the `schemas/` directory is empty after installing from the extension web page.
